### PR TITLE
changed xmllint to jing

### PR DESCRIPTION
**Pull request by schubi2**

### CODE REVIEW

File: xml/ay_bigfile.xml [hunk 1]
```diff
@@ -533,6 +525,22 @@ exit 0
     closed, <command>xmllint</command> will report the errors.
    </para>
 
+
+   <para>
+    For a deeper look e.g. you would like to check if a tag is in the right
+    section or if some tags are missed, the utility <command>jing</command>
```
Comment (imobachgs) [inline xml/ay_bigfile.xml]: missed -> missing ?

**Comment by imobachgs** [comment]
In general, it looks good. I would add a tip (you can use the `<tip>` element) to warn SLE users that `jing` is included in the `SDK` (I'm not sure at 100% but we should check whether is true).

File: xml/ay_bigfile.xml [hunk 2]
```diff
@@ -556,6 +556,13 @@ exit 0
    </para>
 
    <tip>
+    <title>jing</title>
```
Comment (imobachgs) [inline xml/ay_bigfile.xml]: np: what about `Where to find the <literal>jing</literal> package?`

**Comment by imobachgs** [comment]
LGTM, please, merge :)

File: xml/ay_bigfile.xml (hunk 2 above)
Comment (keichwa) [inline xml/ay_bigfile.xml]: Imobach González Sosa <notifications@github.com> writes:

>> @@ -556,6 +556,13 @@ exit 0
>>     </para>
>>  
>>     <tip>
>> +    <title>jing</title>
>
> np: what about `Where to find the <literal>jing</literal> package?`

Or <package>jing</package> --which is available with DB 5 and even
GeekoDoc.

-- 
Karl Eichwalder            SUSE Linux GmbH
R&D / Documentation        Maxfeldstraße 5
                           90409 Nürnberg, Germany
GF: Felix Imendörffer, Jane Smithard, Graham Norton, HRB 21284 (AG Nürnberg)

**Pull request merged**